Tests/Approval.Tests/Approval.Tests.csproj

```diff
@@ -6,14 +6,14 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
-    <PackageReference Include="xunit" Version="2.9.2" />
+    <PackageReference Include="xunit" Version="2.9.3" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.8.2">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="PublicApiGenerator" Version="11.1.0" />
+    <PackageReference Include="PublicApiGenerator" Version="11.3.0" />
```

Comment:
If we [wait a bit](https://github.com/PublicApiGenerator/PublicApiGenerator/issues/437#issuecomment-2580950478) with this upgrade, the generated API shouldn't contain additional parentheses.